lib/App/DuckPAN/Query.pm

```diff
@@ -2,95 +2,125 @@ package App::DuckPAN::Query;
 # ABSTRACT: Main application/loop for duckpan query
 
 use Moo;
-
-my $query;
-my $history_path;
-
+use DDG;
+use DDG::Request;
```

Comment:
This is why Travis was able to build DuckPAN before and the tests all happily passed. This means then that our tests don't actually execute anything that requires DDG. Sigh.

We're going to need to fix that sooner or later...